client/main.lua

```diff
@@ -305,7 +304,7 @@ end
 
 local function CreateZone(x, y, z)
     CreateThread(function()
-        PZone = CircleZone:Create(vector3(x, y, z), 15.0, {
+        PZone = CircleZone:Create(vector3(x, y, z), 15.0, { -- luacheck: ignore
```

Comment:
Remove the ignore comment here as we need that lint error to remind ourselves to change this to ox zones